### PR TITLE
Feature/config color scheme

### PR DESCRIPTION
**Pull request by ajshedivy**
## Explain the reasoning for this pull request. For instance, is it for a new feature, bug fix, code style/cleanup, or something else? If fixing an open issue, please link to it here.

solution for suggestion in #190 for configuring terminal colors 

## Any additional comments/context?

To enable custom terminal colors, head to `/QOpenSys/etc/sc/conf` and open `scrc` with your favorite text editor. Add the following argument with a list of contexts and colors to change. 

```
# Terminal Color defaults used in sc:
#   RUNNING: GREEN
#   NOT_RUNNING: PURPLE
#   INFO: CYAN
#   WARNING: YELLOW
#   ERROR: BRIGHT_RED
#   PLAIN: WHITE
#   STATUS: BLUE
#
# To override these colors, uncomment the following and
# add comma separated list of contexts and colors to change:
#
# --color-scheme=[CONTEXT:COLOR], [CONTEXT:COLOR]

--color-scheme=NOT_RUNNING:RED, RUNNING:CYAN

```
image

### CODE REVIEW

**Comment by chrjorgensen** [comment]
@ajshedivy Thanks for the PR!

One comment:
You have too many commits not relevant to the feature for configuring terminal colors. I think you have based your feature branch on a branch other than the main branch.
Please isolate the commits for terminal color, e.g. create a new feature branch only including commit https://github.com/ThePrez/ServiceCommander-IBMi/pull/195/commits/a820b5b4a6fdd351ce808f486fcbd1ebeb38d9a5 and https://github.com/ThePrez/ServiceCommander-IBMi/pull/195/commits/f69b1ad993c74d947a4401591392bd21a12e041f.

**Comment by ajshedivy** [comment]
Oops ! I I'll go ahead and fix that

File: Makefile [hunk 1]
```diff
@@ -76,6 +76,7 @@ install: scripts/sc scripts/scinit scripts/sc_install_defaults target/sc.jar tar
 	install -m 444 -o qsys target/sc.jar ${INSTALL_ROOT}/QOpenSys/pkgs/lib/sc/sc.jar
 	install -m 555 -o qsys target/scbash ${INSTALL_ROOT}/QOpenSys/pkgs/lib/sc/native/scbash
 	install -m 555 -o qsys target/screbash ${INSTALL_ROOT}/QOpenSys/pkgs/lib/sc/native/screbash
+	dos2unix ${INSTALL_ROOT}/QOpenSys/pkgs/bin/sc
```
Comment (chrjorgensen) [inline Makefile:79]: This should not be necessary - please check my comment in PR #192

**Comment by chrjorgensen** [comment]
@ajshedivy I'll review your upcoming fix tomorrow (I'm in CET) - or @ThePrez could do it if available?

**Comment by ajshedivy** [comment]
Not sure what the best approach is here, I can create a new PR with the correct commits on a new branch.

**Comment by chrjorgensen** [comment]
Yes, create a new branch - the name is not that important, it will only live until approved and merged into main.

**Comment by ajshedivy** [comment]
Okay, let's move discussion to #196, I think that should look better.

**Comment by chrjorgensen** [comment]
I'll review it tomorrow. Thanks!

**Pull request closed without merging**